exonum/src/storage/proof_map_index/mod.rs

```diff
@@ -494,7 +495,7 @@ where
     }
 
     // Inserts a new node as child of current branch and returns updated hash
-    // or if a new node has more short key returns a new key length
+    // or if a new node has a shorter key, returns a new key length
```

Comment:
a new node of the current branch... or, if a new node has a shorter key, returns a new key length.